pkgs/by-name/ce/centerpiece/package.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, pkg-config
+, dbus
+, vulkan-loader
+, libGL
+, fetchFromGitHub
+, rustPlatform
+, libxkbcommon
+, wayland
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "centerpiece";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "friedow";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-I630XrmyRAjVxFvISo2eIUP3YmivZovnV89Xsx5OduY=";
+  };
+
+  cargoSha256 = "sha256-yvvMe1zBUREqRzp/0zYsu7AoXS9Jqq67DY5uMParhEs=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [
+    dbus
+    libGL
+    libxkbcommon
+    vulkan-loader
+    wayland
+  ];
+
+  postFixup = lib.optional stdenv.isLinux ''
+    rpath=$(patchelf --print-rpath $out/bin/centerpiece)
+    patchelf --set-rpath "$rpath:${
+      lib.makeLibraryPath [
+        libGL
+        libxkbcommon
+        vulkan-loader
+        wayland
+      ]
+    }" $out/bin/centerpiece
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/friedow/centerpiece";
+    description = "Your trusty omnibox search";
```

```suggestion
    description = "A blazing fast, extendable launcher for wayland";
```

This looks better imo

---

Upstream chose this description themselves: https://github.com/friedow/centerpiece/blob/d69a20868c33195814166df7e7e70de7e9db21f2/flake.nix#L88

@friedow Which description would you prefer?

---

"Your trusty omnibox search" if you don't mind :)